iquip/apps/builder.py

```diff
@@ -381,6 +376,134 @@ def get_procdesc(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
         }
         return procdesc
 
+    def value(self) -> Dict[str, Any]:
+        """Overridden.
+        
+        Returns a dictionary of scannable arguments from _ScanEntry.
```

Comment:
I think "from _ScanEntry" is unnecessary because this method is in that class.

How about "Returns the dictionary about the selected scannable argument"?